src/util/triggerversion.ts

```diff
@@ -9,15 +9,14 @@ import { Command, getStderrString } from '../tkn';
 import { TektonItem } from '../tekton/tektonitem';
 
 
-export async function newElSupport(): Promise<boolean> {
+export async function triggerSupport(): Promise<string> {
   const result = await TektonItem.tkn.execute(Command.printTknVersion(), process.cwd(), false);
```

Comment:
I'm just wondering, maybe we can create one method/function which calls of `tkn` version and returns some object which contains `tkn` version, Pipeline version and Triggers version, and reuse that object in other checks.